rebar.config

```diff
@@ -1,4 +1,11 @@
 {deps, [ {supervisor3, "1.1.5"}
        , {kafka_protocol, "1.0.0"}
        ]}.
-{erl_opts, [warn_unused_vars,warn_shadow_vars,warn_unused_import,warn_obsolete_guard,debug_info]}.
+{erl_opts, [
+		   warn_unused_vars,
+		   warn_shadow_vars,
+		   warn_unused_import,
+		   warn_obsolete_guard,
+		   debug_info,
+		   {platform_define, "17|18|19", old_rand_uniform}
```

Comment:
if this is only for the deprecated `crypto:rand_uniform` I'd say don't add compile flags since that is used only in tests. simply take the system time's micro part is good enough for testing.